src/gen/lib/AST_grammar.ml

```diff
@@ -1,5 +1,5 @@
 (*
```

Comment:
This file was split into type definitions (still `AST_grammar`) and converter code (`AST_grammar_conv`).